src/nitclk/templates/_library_interpreter.py/_get_error_description.py.mako

```diff
@@ -0,0 +1,15 @@
+    def get_error_description(self, error_code):
+        '''get_error_description
+
+        Returns the error description.
+        '''
+        try:
+            '''
+            It is expected for _get_error to raise when the session is invalid
+            (IVI spec requires GetError to fail).
+            Use _error_message instead. It doesn't require a session.
```

Comment:
It seems we didn't update this comment when we copied this code from the driver templates.
I think we can just delete the comment.

Reply:
Done